CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.19.0)
+cmake_minimum_required(VERSION 3.18.0)
```

Comment:
3.19 was set for interface library feature. It is used for MSVC. Did you test this?
https://cmake.org/cmake/help/latest/command/add_library.html#interface-libraries

Reply:
Didn't mean to downgrade - will revert (i only had 3.18 locally)